website/pages/faq.md

```diff
@@ -0,0 +1,183 @@
+# Frequently Asked Questions
+<sup>(originally by [Relados](https://www.smogon.com/forums/members/relados.148715/) and [Zarel](https://www.smogon.com/forums/members/zarel.102803/); updated by [Blitzamirin](https://www.smogon.com/forums/members/blitzamirin.101585/), [Kalalokki](https://www.smogon.com/forums/members/kalalokki.199661/), Kasumi, [shnen](https://www.smogon.com/forums/members/shnen.107178/), Sobi, [Finland](https://www.smogon.com/forums/members/finland.517429/), [Vacate](https://www.smogon.com/forums/members/vacate.189371/) and [verbatim](https://www.smogon.com/forums/members/verbatim.93103/); converted to Markdown for the PS! website by [Annika](https://www.smogon.com/forums/members/annika.434112/))</sup>
+
+## Staff and Disciplinary
+
+### How do I become staff?
+---
```

Comment:
This seems like the wrong place for a divider bar. I would say no bars at all, but if you wanted to add them, they should be between questions rather than midway through them

Reply:
Oh, sure